data/evenementen/kalender/2025-09-25-eurobsdcon-2025.yaml

```diff
@@ -0,0 +1,19 @@
+id: 2025-09-25-eurobsdcon-2025
+date_start: 2025-09-25
+date_end: 2025-09-28
+event_address:
+  street_address: 
+  postal_code: 
+  city: Dublin
+  country: IE
```

Comment:
This is wrong, the location is Zagreb, Croatia for 2025.
Dublin was this year 2024